Whiskey/Whiskey.psd1

```diff
@@ -105,6 +105,7 @@
                             'Resolve-WhiskeyPowerShellModule',
                             'Resolve-WhiskeyTaskPath',
                             'Resolve-WhiskeyVariable',
+                            'Set-WhiskeyBuildStatus',

```

Comment:
Why did we make this public?

Reply:
It has always been public. That's the old deprecated build status function that I had initially removed and replaced with the `PublishBuildStatusToBitbucket` task. It got re-added when I reverted my commit that completely removed the old `Set-WhiskeyBuildStatus` function in favor of using the new task for the deprecated functionality.